assets/js/util/tracking/createInitializeSnippet.js

```diff
@@ -5,26 +5,30 @@ import createDataLayerPush from './createDataLayerPush';
 import { SCRIPT_IDENTIFIER, DATA_LAYER } from './constants';
 
 /**
- * Returns a function which, when invoked enables tracking and injects the gtag script if necessary.
+ * Returns a function which, when invoked injects the gtag script if necessary.
  *
- * @since 1.3.0
+ * @since n.e.x.t
  *
  * @param {Object} config          Tracking configuration.
  * @param {Object} dataLayerTarget Data layer parent object.
- * @return {Function} Function that tracks an event.
+ * @return {Function} Function that injects gtag script if it isn't yet present.
  */
-export default function createEnableTracking( config, dataLayerTarget ) {
+export default function createInitializeSnippet( config, dataLayerTarget ) {
```

Comment:
@ivankruchkoff this function should return another function as it was before. See Evan's comment: https://github.com/google/site-kit-wp/pull/4142#discussion_r718192118

Reply:
https://github.com/google/site-kit-wp/pull/4142/commits/c8a217c326d5a3a80c84e005c93910da4dd7625b